### DISCUSSION TITLE
Copy URL of a section

### DISCUSSION THREAD

**Discussion by shivaraj-bh** [srid/emanote#478] [Ideas]
For example, [this](https://nixos.asia/en/tutorial/nixos-install) tutorial has an `edit` [section](https://nixos.asia/en/tutorial/nixos-install#edit) and there doesn't seem to be a way to copy the link for it.

**Comment by srid** [comment]
Duplicate of https://github.com/srid/emanote/discussions/137

Also see https://github.com/srid/emanote/issues/480
